lib/rules/display-name.js

```diff
@@ -14,7 +14,9 @@ module.exports = Components.detect(function(context, components, utils) {
 
   var sourceCode = context.getSourceCode();
   var config = context.options[0] || {};
-  var acceptTranspilerName = config.acceptTranspilerName || false;
+  var acceptTranspilerName = config.hasOwnProperty('acceptTranspilerName') ?
```

Comment:
instead of making "absence" and "falsy" be different, would it be better as a breaking change to rename the option to something like "ignoreTranspilerName"?